quinn-proto/src/connection/streams.rs

```diff
@@ -21,6 +21,7 @@ use crate::{
     Dir, Side, StreamId, TransportError, VarInt, MAX_STREAM_COUNT,
 };
 
+#[doc(hidden)]
```

Comment:
Do we actually need these annotations, if the reexports are guarded by `cfg(fuzzing)`?

Reply:
I believe that is correct, and these annotations are added only to satisfy clippy. 

Without them it raises warnings on missing docs, i.e.
```bash
error: missing documentation for a struct
  --> quinn-proto/src/connection/streams.rs:25:1
   |
25 | pub struct Streams {
   | ^^^^^^^^^^^^^^^^^^
   |
   = note: `-D missing-docs` implied by `-D warnings`

error: missing documentation for an associated function
   --> quinn-proto/src/connection/streams.rs:78:5
    |
78  | /     pub fn new(
79  | |         side: Side,
80  | |         max_remote_uni: VarInt,
81  | |         max_remote_bi: VarInt,
...   |
117 | |         this
118 | |     }
    | |_____^

etc ... 
```

Perhaps instead we add an ignore for missing docs here?

Reply:
Yeah, if that's the reason than doing `allow(missing_docs)` makes more sense here.

Reply:
very good thanks that sounds fine to me. updated